From 941712946805db431e846247a3aae70d221bb5f8 Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Wed, 10 May 2023 13:02:16 -0700
Subject: [PATCH 7/7] imx8mn_var_som: Increase CONFIG_SYS_BOOTM_LEN to 64MB

The kernel size in LmP exceeds 32Mb. Increase the maximum size
of an uncompressed kernel to fix the following error:
    Uncompressing Kernel Image
    Error: inflate() returned -5
    gzip compressed: uncompress error -1
    Must RESET board to recover

Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
[mike@foundries.io: ported to imx8mn-var-som]
Signed-off-by: Michael Scott <mike@foundries.io>
---
 include/configs/imx8mn_var_som.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/imx8mn_var_som.h b/include/configs/imx8mn_var_som.h
index 6f55453a642..f2b312b5120 100644
--- a/include/configs/imx8mn_var_som.h
+++ b/include/configs/imx8mn_var_som.h
@@ -17,7 +17,7 @@
 #define CONFIG_CSF_SIZE			0x2000 /* 8K region */
 #endif
 
-#define CONFIG_SYS_BOOTM_LEN		(32 * SZ_1M)
+#define CONFIG_SYS_BOOTM_LEN		(64 * SZ_1M)
 #define CONFIG_SPL_MAX_SIZE		(208 * 1024)
 #define CONFIG_SYS_MONITOR_LEN		(512 * 1024)
 /*
-- 
2.40.1

